index.ts

```diff
@@ -20,15 +18,20 @@ type BuildOutput = {
   code?: string;
 };
 
+// Very partial type - should maybe use the actual TransformOptions type export from Jest's definitions
+type TransformOptions = {
+  transformerConfig: {}
```

Comment:
The Record type seems more appropriate.
```ts
type TransformOptions = {
  transformerConfig: Record<string, any>
}
```

Reply:
@AkifumiSato I went ahead and tried to improve the types further by adding `esbuild` and `@jest/transform` to `devDependencies` so that their types can be used instead of the partial custom ones. Let me know what you think.

`README.md` updated as well.

Reply:
That sounds great!
I think it's the best.